gunicorn/config.py

```diff
@@ -1309,6 +1309,24 @@ class LogConfig(Setting):
     """
 
 
+class LogConfigDict(Setting):
+    name = "logconfig_dict"
+    section = "Logging"
+    cli = ["--log-config-dict"]
```

Comment:
This doesn't seem to work, as there doesn't seem to be any conversion from the string supplied by the shell in some `ARGV[x]` to an actual dict:
```
bash-3.2$ env/bin/gunicorn --version
env/bin/gunicorn --version
gunicorn (version 19.9.0)
bash-3.2$ env/bin/gunicorn --log-config-dict {} 'flask.main:app'
env/bin/gunicorn --log-config-dict {} 'flask.main:app'

Error: Value is not a dictionary: {} 
```

Reply:
Right, maybe `cli` should have been left off this option until someone JSON-parses it.  Can you use config file?

Also https://stackoverflow.com/a/18609361/452210 re `argparse` capabilities in this area.

Reply:
I think we can extend our config we're passing via `--config=python:cariumlib.djangolib.gunicorn_config`
It's funny, I tried hacking `type=json.loads` in, but couldn't get it to work.

Reply:
Maybe you could convert your PR comment into a new issue along the lines of advertised CLI option `--log-config-dict` cannot work.

Reply:
@javabrett Done: https://github.com/benoitc/gunicorn/issues/1909